bundles/org.openhab.binding.flicbutton/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="flicbutton"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="flicd-bridge">
+		<label>FlicButton Bridge</label>
+		<description>This bridge represents a running instance of the fliclib-linux-hci server (flicd).</description>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Flic daemon (flicd) hostname</label>
```

Comment:
```suggestion
				<label>Flic Daemon Address</label>
```
Short and all words start with a capital. IP and hostnames are supported? if so address seems to fit both cases.

Reply:
Sorry, I'll  keep the current version. `flicd` should be mentioned so that people directly understand that the `flicd` service is meant.